### DISCUSSION TITLE
while google login i am getting 429 (Too Many Requests)

### DISCUSSION THREAD

**Discussion by dhanushrajbsetec** [cuongdevjs/reactjs-social-login#157] [Q&A] [answered]
**i am  getting this kind of error for 3 days** 
**what is the problem and how to solve it**

Access to fetch at 'https://cors.bridged.cc/https://www.googleapis.com/oauth2/v3/userinfo?alt=json' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
reactjs-social-login.modern.js:1 
        
        
       GET https://cors.bridged.cc/https://www.googleapis.com/oauth2/v3/userinfo?alt=json net::ERR_FAILED 429 (Too Many Requests)
(anonymous) @ reactjs-social-login.modern.js:1
(anonymous) @ reactjs-social-login.modern.js:1
nr.i @ client:285
(anonymous) @ client:280
LoginRegister.tsx:295 TypeError: Failed to fetch
    at reactjs-social-login.modern.js:1:1
    at nr.callback (reactjs-social-login.modern.js:1:1)
    at nr.i (client:285:46)
    at client:280:329

**Comment by cuongdevjs** [comment]
you're using it in the local environment, don't you?

**Reply by dhanushrajbsetec** [reply]
yes , is it a problem !!?

**Reply by cuongdevjs** [reply, marked as answer]
yes, you can get your own bridge cors's key by yourself, try to follow this instruction [here](https://github.com/gridaco/cors.sh/issues/7)
